mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/converter/marc/Marc4jConverter.java

```diff
@@ -84,7 +84,15 @@ private List<MarcIndicator> convertIndicators(DataField field, Reference referen
 
   private MarcIndicator toMarcIndicator(Reference fieldReference, char indicatorValue, int indicatorIndex) {
     var reference = Reference.forIndicator(fieldReference, indicatorIndex);
-    return new MarcIndicator(reference, indicatorValue);
+    return new MarcIndicator(reference, toIndicatorCommonEmptyValue(indicatorValue));
+  }
+
+  private char toIndicatorCommonEmptyValue(char value) {
+    return switch (value) {
+      // handle empty space and space (32), slash (47) and backslash (92)
+      case 32, 47, 92 -> '#';
```

Comment:
It should be just space.
Please also create a ticket and fix it in mod-quick-marc where quick-marc record is converted to MarcRecord

Reply:
fixed